composer.json

```diff
@@ -14,7 +14,8 @@
         "ext-json": "*",
         "paragonie/sodium_compat": "^1.13",
         "symfony/validator": "^5.1",
-        "guzzlehttp/guzzle": "^7.2"
+        "guzzlehttp/guzzle": "^7.2",
+        "myclabs/deep-copy": "^1.10"
```

Comment:
Figured this was better than writing our own it. It is already required by phpunit but this makes it non-dev requirement.